### PR TITLE
common quad on billboards and particles

### PR DESCRIPTION
**Pull request by yalania**
Use a common quad for particles and billboard instead of generetic a new one everytime to save memory and loading time

### CODE REVIEW

File: Engine/Component/ComponentBillboard.cpp [hunk 1]
```diff
@@ -22,60 +22,11 @@ ComponentBillboard::ComponentBillboard(GameObject* owner) : Component(owner, Com
...
-}
-
-void ComponentBillboard::CleanUp()
-{
-	glDeleteBuffers(1, &vbo);
-	glDeleteBuffers(1, &ebo);
-	glDeleteVertexArrays(1, &vao);
-}
-
 void ComponentBillboard::InitData()
```
Comment (raulgonzalezupc) [inline Engine/Component/ComponentBillboard.cpp:25]: you need to delete the quad on the destructor

File: Engine/Component/ComponentBillboard.h [hunk 2]
```diff
@@ -31,9 +31,7 @@ class ComponentBillboard : public Component
 
 	ComponentBillboard();
 	ComponentBillboard(GameObject* owner);
-	~ComponentBillboard();
-
-	void CleanUp();
+	~ComponentBillboard() = default;
```
Comment (raulgonzalezupc) [inline Engine/Component/ComponentBillboard.h:34]: you need to delete the quad on the destructor

Comment (yalania) [reply Engine/Component/ComponentBillboard.h:34]: is not needed as I'm not allocating memory for the quad, it is managed on the module

**Review by raulgonzalezupc — APPROVED** [review]

**Review by mariofv — APPROVED** [review]
GJ

**Pull request merged**